sherpa-onnx/csrc/online-model-config.h

```diff
@@ -69,7 +73,8 @@ struct OnlineModelConfig {
         debug(debug),
         model_type(model_type),
         modeling_unit(modeling_unit),
-        bpe_vocab(bpe_vocab) {}
+        bpe_vocab(bpe_vocab),
+        tokens_buf_str("") {}
```

Comment:
```suggestion
        bpe_vocab(bpe_vocab) {}
```

`std::string` is default intialized to empty, so there is no need
to re-initialize it here.